### DISCUSSION TITLE
Latest ASR version on conda?

### DISCUSSION THREAD

**Discussion by powellow** [gaynorr/AlphaSimR#50] [Ideas]
@gaynorr @gregorgorjanc: Would it be possible to make the latest version of ASR available on conda? Version 0.10.0 is already available on the conda-forge channel.  Having the latest version would be quite useful to integrate with pipeline managers for more complicated workflows

**Comment by gregorgorjanc** [comment]
@powellow interesting. How is this actually done? @gaynorr did you upload it there or does conda talk to CRAN?

**Reply by gaynorr** [reply]
I actually didn't do anything to get AlphaSimR on conda. I think they just take the top packages from R automatically. I know you can update the packages manually on conda, but I'm by no means an expert at this.
